js/main.js

```diff
@@ -302,7 +402,7 @@ $('#calculate').click(function () {
       selectHist();
     });
 
-    if ((G.nodes().length <= 500) && (selectedGraph == 'Force Directed Layout')) {
+    if ((G.nodes().length <= 500) && (selectedGraph == 'Force Atlas Layout')) {
```

Comment:
Not a big deal at all, but making a note so I don't forget. Why Force Atlas instead of force directed or another time? The only time I've seen Force Atlas is in Gephi, and from briefly googling it seems like it might be a specific name for a force-direction implementation. But, it makes good sense to avoid "directed" so as not to create confusion with directed edges/directed graph.

Reply:
Maybe just "Force Layout"?

![image](https://user-images.githubusercontent.com/6046784/102517586-d7b08600-4055-11eb-8946-912f2e4598d2.png)

Reply:
Ya I wasn't quite sure what we wanted to use and just went with what was already there to be consistent, but love force layout (it's simpler and correct! woot!). I'll add this in my update 😊 

Also thanks for watching/advising @scottbot!!!